setup.py

```diff
@@ -55,6 +55,7 @@ def read_file(fname):
 
 install_requires = [
     'colorlog',
+    'CherryPy==6.0.2',
```

Comment:
Implemented in https://github.com/learningequality/kolibri/pull/241/commits/ecd56ac5a8f70cc56bfe6c602b6cc1606bbb3b47!